site/components/src/main/java/com/visitscotland/brxm/utils/VSRequestContextProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.visitscotland.brxm.utils;
+
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.core.linking.HstLink;
+import org.hippoecm.hst.core.request.HstRequestContext;
+import org.springframework.stereotype.Component;
+
+import javax.jcr.Node;
+
+@Component
+@NonTestable(NonTestable.Cause.BRIDGE)
+public class VSRequestContextProvider {
+
+    public HstRequestContext getRequestContext() {
+        return RequestContextProvider.get();
+    }
+
+    public HstLink getHstLink(Node node){
+        return getRequestContext().getHstLinkCreator().create(node, getRequestContext());
+    }
+
+    public String getUrl (Node node){
```

Comment:
Small space here!

```suggestion
    public String getUrl(Node node){
```

Also, I think there can be valuable tests added this!